raiden-dapp/src/views/Payment.vue

```diff
@@ -78,21 +78,31 @@
 
       <v-spacer></v-spacer>
 
-      <v-dialog v-model="findingRoutes" max-width="625">
+      <v-dialog v-model="serviceSelection" max-width="625">
         <template #activator="{ on }">
           <action-button
             :enabled="valid"
-            @click="findingRoutes = true"
+            @click="serviceSelection = true"
             :text="$t('payment.pay-button')"
             v-on="on"
             class="payment__pay-button"
           ></action-button>
         </template>
+        <v-card class="payment__route-dialog">
+          <pathfinding-services
```

Comment:
Please, make sure it also works if sdk's config.pfs isn't on auto. You need to check `raiden.config.pfs`, and if it's **not** undefined, you must skip the pathfinding-services dialog and call `findRoutes` directly without providing a `pfs` option.

Reply:
Also, I noticed the routes selection dialog is shown even if there's a single route with `fee=0`, which is the direct route case. You should skip the dialog in this specific case and pass this route to `transfer` directly.

Reply:
Would it be better if I just resolved to `null` or something like that on `findPFS` to indicate this case, instead of rejecting and requiring you to inspect config, even if it's public? Should be easy, but it's important to skip the dialog completely if that's the case.

Reply:
Is the config exposed somehow?

Reply:
Yes, it is, there's a `config` getter, and you can check if `pfs` is undefined. You can only call `findPFS` if so.